.github/workflows/govulncheck.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Go vulnerability check'
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - '*'
+    types:
+      - synchronize
+      - opened
+      - reopened
+      - ready_for_review
```

Comment:
I suggest just using the default types. I find the `ready_for_review particularly annoying.

Reply:
I find it better too, I added this because it's what we have everywhere else. Changing it.